src/tar_check.rs

```diff
@@ -111,17 +111,26 @@ fn tar_check_archive<R: Read>(mut archive: Archive<R>, path_str: &str) {
 		if suid_files.is_empty() {
 			eprintln!("Package {} has no SUID files.", path_str);
 		}
+		eprint!("{}{}, ", "[E]".bold(), "=list executable files");
+		eprint!("{}{}, ", "[L]".bold(), "=list all files");
+		eprint!("{}{}, ", "[T]".bold(), "=run shell to inspect");
+
+		//if has_install {
 		eprint!(
-			"[E]=list executable files, [L]=list all files, \
-			 [T]=run shell to inspect, "
+			"{}=show {}, ",
+			"[I]".bold(),
+			"install file".bold().bright_red()
 		);
-		if has_install {
-			eprint!("{}", "[I]=show install file, ".bright_red().bold());
-		};
-		if !suid_files.is_empty() {
-			eprint!("{}, ", "!!! [S]=list SUID files!!!".bright_red().bold());
-		};
-		eprint!("[O]=ok, proceed. ");
+		//};
+
+		//if !suid_files.is_empty() {
+		eprint!(
+			"{}=list {}, ",
+			"[S]".bold(),
+			"SUID files".bold().bright_red()
+		);
+		//};
```

Comment:
I tried it out now, I think I actually like to keep the old `if` condition above. Without it, my terminal alarms me: "hey, you have both SUID and an installation file at your hands, be sure to review it!"  (see screenshot)

![2022 04 03_10:54:28](https://user-images.githubusercontent.com/1547617/161419940-05721766-7587-43cd-94de-ddc6df3f4b48.png)

However, after <strike>clicking on</strike> checking these warnings, you see `Package packagename-1.0.0-x86_64.pkg.tar.zst has no SUID files.`   That's confusing to me, I like the old conditional display better

Reply:
I'm willing to restore the `if` condition and try it out then, maybe that'll be good to keep everyone satisfied and improve the display :)